watchtower/__init__.py

```diff
@@ -9,23 +11,46 @@
 from botocore.exceptions import ClientError
 
 
-def _idempotent_create(client, method, *args, **kwargs):
+if TYPE_CHECKING:
+    from typing_extensions import Protocol
+
+    class LogClient(Protocol):
+        exceptions: Any
+
+        def create_log_group(self, logGroupName: str) -> None:
+            ...
```

Comment:
Please move all of these type declarations into the code itself. I don't think type declarations should be separated from the code.

Reply:
This is providing some typing for boto3 so there is no "into the code itself", I can remove it if you'd prefer to leave boto3 as `Any` but this is a minimal implementation that will prevent future authors from potentially using something that boto3 doesn't provide.

Reply:
Got it, thanks. Watchtower will not be providing type hints for other libraries, and it will also not use stub type declarations.